ui/components/app/assets/token-list/token-list.tsx

```diff
@@ -85,7 +87,7 @@ export default function TokenList({
         return (
           <TokenCell
             key={`${tokenData.symbol}-${tokenData.address}`}
-            {...tokenData}
+            {...{ ...tokenData, chainId }}
```

Comment:
Rather than double spreading, I think passing in `chainId` as a dedicated prop is a bit cleaner:
```suggestion
            {...tokenData}
            chainId={chainId}
```

Reply:
fixed [0912127](https://github.com/MetaMask/metamask-extension/pull/27724/commits/0912127050ba6e647f334f6be4588634e214d47d)